### ISSUE TITLE
ft.encode_features creates duplicates columns when one category is "unknown"

### ISSUE THREAD

**Issue by WillKoehrsen** [Featuretools/featuretools#280]
When using `ft.encode_features` with a dataframe that has a categorical column with the category `"unknown"`,  duplicate columns are created. 

This can be reproduced with the following code:

```python
import featuretools as ft
import pandas as pd

# Dataframe with categorical column with "unknown" string
df = pd.DataFrame({'category': ['unknown', 'b', 'c', 'd' , 'e']})

# Featuretools
es = ft.EntitySet('test')
es.entity_from_dataframe(entity_id = 'a', dataframe = df, index = 'index', make_index = True)
feature_matrix, feature_defs = ft.dfs(entityset = es, target_entity = 'a')

# Encode the features
features_enc, feature_defs_enc = ft.encode_features(feature_matrix, feature_defs)
```
The resulting `features_enc` dataframe has a duplicated column:

![features_enc dataframe](https://user-images.githubusercontent.com/17484616/46698249-83432d80-cbe4-11e8-80c1-61c75fe296dc.png)

This is an extreme corner case and does not occur with `include_unknown` set to `False`.

**Comment by WillKoehrsen** [comment]
One solution would be to check that the string `"unknown"` does not occur in any categories, and if it does, use a different placeholder. We'd want the same placeholder for all categoricals though so maybe there should be a `placeholder` parameter set by the user with default `"unknown"`.

**Comment by kmax12** [comment]
Fixed by #287  closing
